.github/workflows/docs.yaml

```diff
@@ -5,30 +5,48 @@ on:
       - main
     tags:
       - 'v*'
+  pull_request:
```

Comment:
I added this so we could test the action in this PR … but I'd like to remove it before we merge.